src/main/webapp/chat.js

```diff
@@ -0,0 +1,32 @@
+async function addToChat() {
+  let chatInput = document.getElementById("chat-input").value;
+  let data = {
+    message: chatInput,
+  };
+
+  let response = await fetch("/chat", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(data),
+  });
+}
+
+function updateChat(data) {
+  let newMessage = document.createElement("li");
+  newMessage.appendChild(document.createTextNode(data.message));
+  let chat = document.getElementById("chat");
+  chat.appendChild(newMessage);
+}
+
+Pusher.logToConsole = false;
+
+var pusher = new Pusher("d15fbbe1c77552dc5097", {
+  cluster: "us2",
+});
+
+var channel = pusher.subscribe("spotify-game-app");
```

Comment:
Similar to what I suggested in the Java file, could you add constants for the "spotify-game-app" and "chat-update" strings?

In the future, we'll want to think about whether we can share strings like these across the Java and JavaScript code and declare them in one place. But for now, having them be declared with the same constant names in both the Java and JavaScript code should make it easier to see that they're the same strings used on the both the client and server.